[APPSEC-57941] AAP Go exhaustive docs (#30303)
DataDog/documentation: content/en/security/application_security/setup/go/_index.md

---
title: Enabling App and API Protection for Go
aliases:
  - /security_platform/application_security/getting_started/go
  - /security/application_security/getting_started/go
  - /security/application_security/threats/setup/threat_detection/go
  - /security/application_security/threats_detection/go
further_reading:
- link: "/security/application_security/add-user-info/"
  tag: "Documentation"
  text: "Adding user information to traces"
- link: 'https://github.com/DataDog/dd-trace-go'
  tag: "Source Code"
  text: 'Go Datadog library source code'
- link: "/security/default_rules/?category=cat-application-security"
  tag: "Documentation"
  text: "OOTB App and API Protection Rules"
- link: "/security/application_security/troubleshooting"
  tag: "Documentation"
  text: "Troubleshooting App and API Protection"
---
{{< partial name="app_and_api_protection/callout.html" >}}

## Overview

> App and API Protection leverages the [Datadog Go library][5] to monitor and secure your Go service. The library integrate seemlessly into your workflow using [Orchestrion][6] compile-time instrumentation without requiring any code changes.

Reviewer: ```suggestion
App and API Protection (AAP) leverages the [Datadog Go library][5] to monitor and secure your Go service. The library integrates seamlessly into your workflow using [Orchestrion][6], an automatic compile-time instrumentation of Go code that does not require code changes. 
```
